core/tests/webdriverio_utils/SkillEditorPage.js

```diff
@@ -234,6 +241,23 @@ var SkillEditorPage = function() {
     expect(await saveOrPublishSkillButton.isEnabled()).toEqual(false);
   };
 
+  this.saveChangesToQuestion = async function(commitMessage) {
+    await general.scrollToTop();
+    await action.click('Save Question Button', saveQuestionButton);
+    await action.setValue('Commit message', commitMessageField, commitMessage);
+    await action.click('Close save modal button', closeSaveModalButton);
+    await waitFor.invisibilityOf(
+      closeSaveModalButton, 'Save modal takes too long to disappear.');
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.expectQuestionInteractionIdToMatch = async function(interactionId) {
```

Comment:
This should be expectedInteractionId

Reply:
I tried to follow the syntax of the already existing 'expect' methods. If you want then I can change it.

Reply:
Ah, hm, it does seem repeated all over the place here. Yeah, I think it's fine, let's leave as-is. Thanks!